mmcls/models/backbones/tnt.py

```diff
@@ -0,0 +1,344 @@
+import math
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import build_norm_layer
+from mmcv.cnn.bricks.transformer import FFN, MultiheadAttention
+from mmcv.cnn.utils.weight_init import trunc_normal_
+from mmcv.runner.base_module import BaseModule, ModuleList
+
+from ..builder import BACKBONES
+from ..utils import to_2tuple
+from .base_backbone import BaseBackbone
+
+
+class TnTLayer(BaseModule):
+    """Implement one encoder layer in Transformer in Transformer.
+
+    Args:
+        inner_dims (int): The feature dimension in inner transformer block
+        embed_dims (int): The feature dimension in outer transformer block
```

Comment:
`inner_dims` and `embed_dims` are a little weird.
`inner_embed_dims` and `outer_embed_dims` may be better.